atlite/convert.py

```diff
@@ -164,7 +164,7 @@ def convert_and_aggregate(cutout, convert_func, windows=None, matrix=None,
     if capacity_factor:
         assert aggregate_func is aggregate_sum, \
             "The arguments `matrix`, `shapes` and `layout` are incompatible with capacity_factor"
-        results /= len(cutout.meta['time'])
+        results /= cutout.data.coords['time'].size
```

Comment:
```suggestion
        results /= cutout.data['time'].size
```

Reply:
The `coords` indirection is not useful in this case. There is no clear case for `len` or `.size`, so I don't care.

Reply:
Does the suggestion work here? Don't know what the conditions are.

Reply:
Bump! Can you apply the modification above?

Reply:
Sure, done. Thanks for bumping.
This kind of went under my radar.